feat: Make clear association between incidents and deployments (#71)
codacy/pulse-user-docs: docs/metrics/accelerate.md

    As such, Pulse considers the **deployment** that caused an **incident** the closest deployment before the start of that **incident**. The same deployment might be responsible for multiple incidents.

>     Pulse calculates the performance metrics per system and later aggregates the metrics by time interval when displaying them. This means that you need both deployments and incidents for Pulse to correctly map the system between the two types of events and for the metrics to work.

Reviewer: I "ate" :yum: a blank line in my previous suggestion, sorry:

```suggestion
    Pulse calculates the performance metrics per system and later aggregates the metrics by time interval when displaying them. This means that you need both deployments and incidents for Pulse to correctly map the system between the two types of events and for the metrics to work.
    
```

    As such, Pulse considers the **deployment** that caused an **incident** the closest deployment before the start of that **incident**. The same deployment might be responsible for multiple incidents.

    Pulse calculates the performance metrics per system and later aggregates the metrics by time interval when displaying them. This means that you need both deployments and incidents for Pulse to correctly map the system between the two types of events and for the metrics to work.

>     

Author: Codacy has a fix for the issue: [Expected: 0 or 2; Actual: 4](https://app.codacy.com/gh/codacy/pulse-user-docs/pullRequest?prid=7914109)
```suggestion

```

Reviewer: Uuuh, it's the first time I see this working! :star_struck:

>     As such, Pulse considers the **deployment** that caused an **incident** the closest deployment before the start of that **incident**. The same deployment might be responsible for multiple incidents.
> 
>     When viewing the performance of specific systems, both incidents and deployments will be first filtered, and only then associated.
> 

Reviewer 2: I am not sure if it got much better, but I for me it is a bit more clear.

```suggestion
    As such, Pulse considers the **deployment** that caused an **incident** the closest deployment before the start of that **incident**. The same deployment might be responsible for multiple incidents.

    When viewing the performance, it is calculated per system before being aggregated by the time buckets. This also means that you need the deployments and incidents to have the right system association for the metrics to work.

```

>     When viewing the performance, it is calculated per system before being aggregated by the time buckets. This also means that you need the deployments and incidents to have the right system association for the metrics to work.
> 

Reviewer: "Time buckets" is internal lingo that customers may not understand. Can we use "time intervals" instead?

Also, here's my go at trying to improve the readability of this information by making some details more explicit:

```suggestion
    Pulse calculates the performance metrics per system and later aggregates the metrics by time interval when displaying them. This means that you need both deployments and incidents for Pulse to correctly map the system between the two types of events and for the metrics to work.
```

Author: 🎖️

Reviewer 2: Intervals sounds weird to me since that is the other thing in the filters. The buckets are the aggregation intervals at most.

Reviewer: OK, so maybe we can call them "time aggregation intervals" - or even "time aggregation buckets"? At least for me, this already clarifies better how things work. :ok_hand:

>     

Author: Codacy has a fix for the issue: [Expected: 0 or 2; Actual: 4](https://app.codacy.com/gh/codacy/pulse-user-docs/pullRequest?prid=7914109)
```suggestion

```
